test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -929,3 +929,68 @@ def test_suspended_job_gets_calendered(self):
         self.scheduler.log_match(
             jid1 + ";Can't find start time estimate", existence=False,
             max_attempts=2)
+
+    def helper_test_preempt_release_all(self, preempt_method):
+        """
+        Helper function to test that when preempting jobs, all resources
+        are released during preemption simulation for R and C methods
+        """
+        if preempt_method == "R":
+            schedlog_msg = "Job preempted by requeuing"
+        elif preempt_method == "C":
+            schedlog_msg = "Job preempted by checkpointing"
+        else:
+            raise Exception("Unexpected value of argument preempt_method: %s"
+                            % (preempt_method))
+
+        a = {ATTR_restrict_res_to_release_on_suspend: 'mem'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        self.scheduler.set_sched_config({'preempt_order': preempt_method})
+
+        # Set 1gb mem available on the node
+        a = {ATTR_rescavail + '.ncpus': "2"}
```

Comment:
You are not setting memory on the node

Reply:
That's fine, it doesn't matter since the jobs don't request memory